.github/workflows/run-docker.yml

```diff
@@ -10,8 +10,29 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - name: Build Docker image
-        run: docker build -f docker/jupyter.Dockerfile -t traffic/jupyter docker
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Import the library
-        run: docker run traffic/jupyter python -c "import traffic; print(traffic.__version__)"
+      # jupyter Docker image for users
+      - name: Build jupyter Docker image
+        run: docker build --file docker/jupyter.Dockerfile --tag ghcr.io/xoolive/traffic/jupyter:latest docker
+
+      - name: Import the library for testing
+        run: docker run ghcr.io/xoolive/traffic/jupyter:latest python -c "import traffic; print(traffic.__version__)"
```

Comment:
We could use that command to pick the version number and include it in the tag version, no?

Reply:
The release tag is now taken from the build trigger:
![grafik](https://user-images.githubusercontent.com/44362748/180744819-c97a0139-814c-4b74-8cb9-e23d2088cd5f.png)

If you have a different workflow in mind, feel free to change it to your taste.

Reply:
oh wow, nice! Didn't know we could do that!